docker-compose.yml

```diff
@@ -99,9 +75,9 @@ services:
       dockerfile: Dockerfile.react
     command: npm run dev
     depends_on:
-      - dev_backend
+      - django
     links:
-      - dev_backend:drf
+      - django:drf
```

Comment:
@iacobfred we can remove this link now that the underscore is gone. The api link in [`frontend/pages/entities/index.js`](https://github.com/ModularHistory/modularhistory/blob/b4b65ab55d4a7557165867bae3c8027693bfc6c3/frontend/pages/entities/index.js#L104) will need to be updated. We should create a new environment variable for the api root (`http://django:8001/api/`).